examples/alpine/basic-dynamic-columns/src/index.css

```
@@ -0,0 +1,131 @@
+  user-select: none;
+}
+
+.scroll-container {
+  overflow-x: auto;
+}
+
+.demo-button,
+.page-size-input,
+.filter-input,
+.filter-select {
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)